packages/relay/src/lib/services/metricService/metricService.ts

```diff
@@ -0,0 +1,297 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { Logger } from 'pino';
+import { Status } from '@hashgraph/sdk';
+import HbarLimit from '../../hbarlimiter';
```

Comment:
same goes for the import type here

Reply:
https://github.com/hashgraph/hedera-json-rpc-relay/pull/2873#discussion_r1736637552